php/ad-servers/class-ad-layers-ad-server.php

```diff
@@ -70,13 +70,24 @@ class Ad_Layers_Ad_Server extends Ad_Layers_Singleton {
 		public $handle = 'ad-layers';
 
 		/**
+		 * Capability required to manage the ad server settings. Defaults to
+		 * `manage_options`.
+		 *
+		 * @var string
+		 */
+		public $settings_capability = 'manage_options';
```

Comment:
All of the capabilities (not just here) are public and deliberately referenced after `init` so that anyone can change them by modifying the singleton's property. For instance, if someone needed to modify this to give access to anyone with "edit_posts" capabilities, they might do something like...

```php
add_action( 'init', function() {
    Ad_Layers_Ad_Server::instance()->settings_capability = 'edit_posts';
} );
```

Having said that, I understand and agree that this is not a typical WordPress pattern. If you think I should switch these to all be filtered, I'm easily convinced.